code/modules/law/laws/low_severity.dm

```diff
@@ -157,7 +157,7 @@
 
 /datum/law/low_severity/necrophilia
 	name = "i116 - Disrespect to the Dead"
-	desc = "To threaten to kill or seriously injure an employee."
```

Comment:
The law datums are only relevant if we arnt pulling the "laws" from the db.
Which we are. So the law datums are irrelevant.

Reply:
Then the DB is also incorrect as it has this same issue. Regardless it fixes something in the code.